lists/list.css

```diff
@@ -0,0 +1,132 @@
+@import url('https://fonts.googleapis.com/css2?family=Fredoka&display=swap');
+
+:root{
+    --primary-color: #34D399;
+}
+
+
+body{
+    margin: 0 auto;
+    font-family: 'Fredoka', sans-serif;
+}
```

Comment:
The CSS styles written here most probably will apply to all your components. If this is getting repeated in other components as well, you can extract it into a different file and then import it wherever this code repeats.